recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
```

Comment:
This is what I found in the docker image when inspecting the `libgfortran` library.